.gitignore

```diff
@@ -29,6 +29,7 @@ mtest_opponent.exe
 # ignore eclipse project files
 .cproject
 .project
+.settings
```

Comment:
this also please as a separate commit

Reply:
> this also please as a separate commit

Why? I can make a separate commit for 1.2.1 anyway, together with the MP_PRIVATE change above.

Reply:
Because there's `cherry-pick -x` to do something like that, not a new commit.

Reply:
Simply leave it as is, I'll go for the separate commit